### PR TITLE
Tornar buildpack funcional com Go

### PR DESCRIPTION
**Pull request by Mahezerlabs**
Desconsiderem os commits, hehe...

Eu precisava subir no git pro teresa baixar e encontrar...

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,20 +12,24 @@ In order for this buildpack to execute, it will look for a `.oracle.yml` file in
...
+This buildpack will need to be invoked first, followed by your language buildpack.  
+Heroku now supports configuring multiple buildpacks natively, or you can use the [heroku-buildpack-multi](https://github.com/ddollar/heroku-buildpack-multi) buildpack.
 
 ### Setup using heroku-buildpack-multi
 
 Add the following contents to `.buildpacks`:
 
-    https://github.com/luizalabs/oracle-heroku-buildpack
-    https://github.com/heroku/heroku-buildpack-python
+    https://github.com/Mahezerlabs/oracle-heroku-buildpack
```
Comment (drgarcia1986) [inline README.md]:
```diff
- https://github.com/Mahezerlabs/oracle-heroku-buildpack
+ https://github.com/luizalabs/oracle-heroku-buildpack
```

File: bin/compile [hunk 2]
```diff
@@ -53,37 +52,42 @@ fi
...
 mkdir -p ${BUILD_DIR}/.profile.d
 
+ORACLE_HOME="${HOME}/${ORACLE_INSTANT_CLIENT_DIR}"
+
 cat <<EOF > ${BUILD_DIR}/.profile.d/oracle.sh
   export OCI_LIB_DIR=${ORACLE_HOME}
   export ORACLE_HOME=${ORACLE_HOME}
   export OCI_INC_DIR=${ORACLE_HOME}/sdk/include
-  export LD_LIBRARY_PATH=${ORACLE_HOME}:\$LD_LIBRARY_PATH}
+  export LD_LIBRARY_PATH=${ORACLE_HOME}
```
Comment (drgarcia1986) [inline bin/compile]: maybe ?
```diff
- export LD_LIBRARY_PATH=${ORACLE_HOME}
+ export LD_LIBRARY_PATH=${ORACLE_HOME}:\$LD_LIBRARY_PATH
```
To preserve the previous `LD_LIBRARY_PATH` value

**Comment by drgarcia1986** [comment]
@Mahezerlabs tks for your PR, I really appreciate.
Could you squash those commits? There are 66 commits for 3 files changed :sweat_smile:

@aguerra could you check this?

**Comment by Mahezerlabs** [comment]
Sure, no problem! :smile:

**Comment by Mahezerlabs** [comment]
Done ;)

**Review by drgarcia1986 — APPROVED** [review]
lgtm

**Review by aguerra — APPROVED** [review]
lgtm

By the way, PKG_CONFIG_PATH seems to be equal to ORACLE_HOME all the time, maybe you don't need it

**Comment by Mahezerlabs** [comment]
Actually, I do need it. The oci8 package (which is installed in the next buildpack) uses it to find the `oci8.pc` file

**Comment by drgarcia1986** [comment]
Thank you @Mahezerlabs

**Pull request merged**